src/MassTransit/Middleware/OutboxMessagePipe.cs

```diff
@@ -54,7 +54,7 @@ public async Task Send(OutboxConsumeContext<TMessage> context)
 
             LogContext.Debug?.Log("Outbox Completed: {MessageId} ({ReceiveCount})", context.MessageId, context.ReceiveCount);
 
-            if (!context.ReceiveContext.IsDelivered && !context.ReceiveContext.IsDelivered)
+            if (!context.ReceiveContext.IsDelivered)
```

Comment:
This will not be changed, the reality is that it needs to be determined if it should be !isFaulted.